ansible/roles/kubernetes/files/make-ca-cert.sh

```diff
@@ -120,8 +125,8 @@ fi
 mkdir -p "$cert_dir"
 
```

Comment:
I would recommend to run ``ls pki/issued pki/private`` to see what is inside the directory.